packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -954,6 +954,17 @@ - (NSDictionary *)getJSError:(NSError *)error {
   NSString *message = [error localizedDescription];
   NSString *nativeErrorMessage = [error localizedDescription];
 
+  if(error.domain == @"FIRAuthErrorDomain"){
```

Comment:
I would also add another condition (e.g. `&& code == 17999` ) as an additional check, but depends on whether it's an acceptable solution.

Reply:
Agreed. Added!